.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
-*.log
 dist
+.idea/
```

Comment:
You should use something like [this](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead. I keep `*.log` in all my `.gitignore`s to catch `yarn-error.log` etc.

Reply:
.idea folder is for the IDE IntelliJ IDEA much like how VSCode created a .vscode folder. There are plenty of projects that I personally work on (heck the company I work for in fact) where the .idea folder does get comitted, it is therefore unrealistic to globally ignore it. 

I could remove it yes but that does end up meaning that if other people fork your project they are constantly going to have to either pass that folder when `git add`ing or delete it entirely before said process.

Reply:
Ok no worries :+1: